.env.example

```diff
@@ -1,30 +0,0 @@
-# Application
```

Comment:
Do not remote `.env.example` file.
This file will let us know what environment specific variables need to be specified in the `.env` file (which you should not push to remote repository).

Reply:
ok dai